src/test/java/org/openelisglobal/address/PersonAddressTest.java

```diff
@@ -0,0 +1,46 @@
+package org.openelisglobal.address;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.address.valueholder.PersonAddress;
+
+public class PersonAddressTest {
+
+    private PersonAddress personAddress;
+
+    @Before
+    public void setUp() {
+        personAddress = new PersonAddress();
+    }
+
+    @Test
+    public void testSetAndGetPersonId() {
+        personAddress.setPersonId("123");
+        Assert.assertEquals("123", personAddress.getPersonId());
+    }
+
+    @Test
+    public void testSetAndGetAddressPartId() {
+        personAddress.setAddressPartId("456");
+        Assert.assertEquals("456", personAddress.getAddressPartId());
+    }
+
+    @Test
+    public void testSetAndGetUniqueIdentifier() {
+        personAddress.setUniqueIdentifyer("unique123");
```

Comment:
`Assert.assertEquals(personAddress.setUniqueIdentifyer("unique123"),  personAddress.getUniqueIdentifyer());`

this too ..

do the rest elsewhere